OpenRA.Mods.Common/Traits/GivesExperience.cs

```diff
@@ -22,6 +22,14 @@ class GivesExperienceInfo : ITraitInfo
 		[Desc("Stance the attacking player needs to receive the experience.")]
 		public readonly Stance ValidStances = Stance.Neutral | Stance.Enemy;
 
+		[Desc("Modifier (in percent) with which the `Experience` value will be multiplied",
```

Comment:
After all the recent misunderstandings I suggest we blacklist any use of the word "multiply" referring to modifiers.

Can we change this to "Percentage of the Experience value that is granted to the killer"?  This is more in line with our descriptions for other modifiers?